tests/feinterface/fe_interface_view_get_function_from_local_dof_values_02.output

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
This output files will need to be called `fe_interface_view_get_function_from_local_dof_values_02.with_trilinos=true.with_trilinos_wiith_sacado=true.output`.

Reply:
that's why the test failed, I think.